File: 0027-remove-element/0027-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        
            int i=0;
            int k=nums.size()-1;
            while(i<=k){
                    while(nums[k]==val){
                            k--;
                            if(k<0){
                                    return 0;
                            }
                    }
                    if(nums[i]==val and i<=k){
                            nums[i]=nums[k];
                            k--;
                    }
                    i++;
            }
            cout<<k<<endl;
            return k+1;
            
    }
};

File: 0027-remove-element/0027-remove-element.py
class Solution:
    
    def removeElement(self, nums: List[int], val: int) -> int:
        
        i=0
        j=len(nums)-1
        
        while(i<=j):
            
           
            if(j>=0 and nums[j]==val):
                        j-=1
                        continue
                        
            if(nums[i]==val):
                    nums[i],nums[j]=nums[j],nums[i]
                    j-=1
                    
            i+=1
                    
           
       
        return j+1
        
        